guides/plugins/plugins/administration/using-base-components.md

```diff
@@ -0,0 +1,38 @@
+# Using base components
+
+The Shopware 6 Administration comes with a bunch of tailored Vue-components, already accessible in all of your templates via the `component registry`.
+This guide will show you how you can use Shopware-made components in your templates, 
+if you want to learn more about the `component registry` and how you can register your own components to it have a look at the [corresponding guide](./add-custom-component.md)
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance, the files and preferably a registered module. Of course you'll have to understand JavaScript and have a basic familiarity with [Vue](https://vuejs.org/), the framework, used in the Administration. However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Finding the base component needed
+
+All Shopware 6 Administration components can be seen in the [Component Library](https://component-library.shopware.com/). 
+There you can see what each of the components does and looks like, it also shows you what props the can work with and wich slots they have.
+
+## Using the base
+
+As mentioned before in the introduction, all components used in the Shopware 6 Administration are registered in the `component registry`, which makes them accessible in the Administration.
+
+Using base components in your own administration templates is rather simple. In the example below we will use the `sw-text-field` in our template, which simply renders a `text` input tag, but also supports some fancy functionality, like inheritance, etc:
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/component/example-component/example.html.twig" %}
+```html
+<div>
+    <sw-text-field />
+</div>
+```
+{% endcode %}
+
+That's basically it. To continue building beautiful custom components, learn how to write templates and how to include them in your components [here](./writing-templates.md).
+
+## Next steps
+
+Now that you know that all components are accessible through the component registry, you might want to learn more about the following things:
```

Comment:
What really confuses me here:
You're talking a lot about the component registry and "things being accessible through the component registry", yet all that's done is "simply using" the component in the templates. This way it feels like there's missing something, some kind of "Component.Registry.getElement". I know this is not a thing, but the guide kinda makes me feel like such a thing is missing.

TL;DR: Maybe at least link the "component registry", if it's a class or something?

Reply:
The `Component.Registry.getElement `thing is approximately what internally happens, when a component is used in a template. i'll write a sentence about it

Reply:
i hope its clearer now